improver/utilities/cube_extraction.py

```diff
@@ -354,9 +357,9 @@ def pressure_grid(variable_on_pressure: Cube) -> np.ndarray:
         return pressure_array
 
     def fill_invalid(self, cube: Cube):
-        """Populate any invalid values with the maximum non-infinite value in that column
-        on the assumption that missing values fall below the model's surface.
-        (The maximum is actually inflated by 0.1 to allow the interpolator to work)."""
+        """Populate any invalid values with the neighbouring value in that column plus the median
+        difference. This results in columns that do not have repeated values, which confuses
+        the stratify.interpolate method."""
```

Comment:
1) I think this might be better written as "This results in columns that do not have repeated values as repeated values confuses the stratify.interpolate method". Otherwise it reads like columns without repeated values confuses stratify.interpoalte 
2) I don't bleieve the method is using the median value anymore

Reply:
Doc-string updated.